scripts/SPM/Sources/Core/Secrets/EnvironmentUpdater.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  EnvironmentUpdater.swift
+//  Basic
+//
+//  Created by Ellen Shapiro on 4/9/19.
+//
+
+import Foundation
+import Files
+
+struct EnvironmentUpdater {
+    enum EnvironmentKey: String, CaseIterable, KeyToUpdate {
```

Comment:
This needs proper documentation. Adding a new entry is not obvious.

Reply:
Wholeheartedly agreed, which is why this is in the list of TODOs in the PR for the *next* PR.  

I wanted to get this shipped first so building properly could happen while I wrote up the docs.